extensions/kubernetes-service-binding/runtime/src/main/java/io/quarkus/kubernetes/service/binding/runtime/KubernetesServiceBindingConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.kubernetes.service.binding.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "kubernetes-service-binding", phase = ConfigPhase.BOOTSTRAP)
+public class KubernetesServiceBindingConfig {
+
+    /**
+     * If enabled, Service Bindings will be looked in the file system
+     */
```

Comment:
Missing `@ConfigItem`? Mostly for generated docs ;-)

Otherwise, looks great. Thanks.

Reply:
Ah darn... Feel free to send a PR and I'll approve right away :)